### PR TITLE
feat: Add browser info and app version

### LINKED ISSUE #529: Add version and diagnostic information, accessible at all routes
To make it easier to report issues, we should supply a visible version number and any other pertinent diagnostic information in the UI.

### PR DESCRIPTION
**Pull request by rodrigolabs**
This PR adds the `Version number` near the OHIF's logo and in the About modal.

Also, it was added the `Build Number`, `Browser` and `OS` in the About Modal.

A new `i18n` namespace was added in Locize and translations for `en-US` and `pt-BR` were  made.
___

### Preview

Screen Shot 2019-10-15 at 1 40 55 PM

Screen Shot 2019-10-15 at 1 37 57 PM

Screen Shot 2019-10-15 at 1 38 33 PM

___
Fixes #529

### CODE REVIEW

File: platform/ui/src/components/userPreferencesModal/AboutModal.js [hunk 1]
```diff
@@ -69,13 +97,15 @@ export class AboutModal extends Component {
             >
               Visit the forum
             </a>
+            {` `}
```
Comment (swederik) [inline platform/ui/src/components/userPreferencesModal/AboutModal.js]: @dannyrb Is this an ESLint thing? Why do these keep getting inserted?

Comment (dannyrb) [reply platform/ui/src/components/userPreferencesModal/AboutModal.js]: It assumes you want to preserve a space. Newline? No space. This thing? A space.

File: platform/ui/package.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@ohif/i18n": "^0.2.3",
+    "browser-detect": "^0.2.28",
```
Comment (dannyrb) [inline platform/ui/package.json]: yarn.lock updates?

Comment (rodrigolabs) [reply platform/ui/package.json]: Already pushed :)

File: .webpack/webpack.base.js [hunk 3]
```diff
@@ -1,5 +1,6 @@
 const path = require('path');
 const webpack = require('webpack');
+const PACKAGE = require('../platform/viewer/package.json');
```
Comment (dannyrb) [inline .webpack/webpack.base.js]: Reminder for myself, if we need to pull from here, then we need to make sure we're doing a build/deploy after a successful lerna version for this to be accurate. I may do this in a separate PR

File: platform/ui/src/components/userPreferencesModal/AboutModal.js [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
 import './AboutModal.styl';
+import { withTranslation } from '../../utils/LanguageProvider';
 
 import 'react-bootstrap-modal/lib/css/rbm-patch.css';
 
 // TODO: Is this the only component importing these?
 import './../../design/styles/common/modal.styl';
 
-export class AboutModal extends Component {
+class AboutModal extends Component {
```
Comment (dannyrb) [inline platform/ui/src/components/userPreferencesModal/AboutModal.js]: Not worth doing for this PR, but simple/dumb components can probably just be Function components

File: platform/viewer/src/components/Header/Header.js [hunk 5]
```diff
@@ -101,6 +101,14 @@ class Header extends Component {
             </Link>
           )}
 
+          <span
+            className={`${
+              this.props.home ? 'header-versionInfoHome' : 'header-versionInfo'
```
Comment (dannyrb) [inline platform/viewer/src/components/Header/Header.js]: We should probably find a better way to do this, but good enough for now.

**Comment by dannyrb** [comment]
LGTM :+1:
